packages/fxa-auth-server/docs/swagger/session-api.ts

```diff
@@ -99,6 +99,40 @@ const SESSION_RESEND_CODE_POST = {
   notes: ['🔒 Authenticated with session token'],
 };
 
+const SESSION_VERIFY_TOKEN_GET = {
+  ...TAGS_SESSION,
+  description: '/session/verify/token',
+  notes: [
+    dedent`
+      🔒 Authenticated with session token
+
+      Verifies the session token and checks various verification states including account verification, session verification, and authenticator assurance level (AAL) compliance. The required AAL is determined from the account's available authentication methods.
+
+      This endpoint returns a response indicating the verification status rather than throwing errors for verification needs.
+    `,
+  ],
+  plugins: {
+    'hapi-swagger': {
+      responses: {
+        400: {
+          description: dedent`
+            Failing requests may be caused by the following errors (this is not an exhaustive list):
+            - \`errno: 110\` - Invalid token
+          `,
+        },
+        200: {
+          description: dedent`
+            Successful responses include verification status information:
+            - \`accountStatus\`: "verified" or "unverified" (never null; if the session token is invalid or the account has been deleted, the endpoint returns an error)
+            - \`sessionStatus\`: "verified", "mustVerify", "mustUpgrade", or null
+            - \`verificationMethod\`: Required verification method (if verification needed)
```

Comment:
Should `verificationMethod` have the possible methods listed (`'totp-2fa'`, `'email-otp'`), to match the other descriptions?

Also when can `sessionStatus` ever be `null`? I see it marked as possibly null everywhere, but don't see where this can be `null` looking at the endpoint.